package/Dockerfile_tests

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.13-alpine
+
+WORKDIR /go/src/github.com/openebs/jiva
+COPY . .
+
+RUN apk add build-base
+
+CMD ["sh", "/go/src/github.com/openebs/jiva/scripts/test"]
```

Comment:
Your thought process behind this approach is very good, but no need to test in container, we can directly test it on the host. So please remove this file.

Reply:
You can create virtual block devices on the host. PTAL  https://superuser.com/questions/1033493/how-to-create-virtual-block-devices-from-file

Reply:
I'm not that experienced with devices, but from I was reading, using this also need to run as `root`.
This still will make the test fail locally or maybe I'm not understanding the solution correctly.

Reply:
Ok, sounds good
cc: @vitta